client/src/components/EmployeeCard/AvailabilityOptions.js

```diff
@@ -0,0 +1,191 @@
+const options = [
+  {
+    label: 'edit'
+  },
+  {
+    label: '12:00 am',
+    value: 0
+  },
+  {
+    label: '12:30 am',
+    value: 0.5
+  },
+  {
+    label: '1:00 am',
+    value: 1
+  },
+  {
+    label: '1:30 am',
+    value: 1.5
+  },
+  {
+    label: '2:00 am',
+    value: 2
+  },
+  {
+    label: '2:30 am',
+    value: 2.5
+  },
+  {
+    label: '3:00 am',
+    value: 3.0
+  },
+  {
+    label: '3:30 am',
+    value: 3.5
+  },
+  {
+    label: '4:00 am',
+    value: 4
+  },
+  {
+    label: '4:30 am',
+    value: 4.5
+  },
+  {
+    label: '5:00 am',
+    value: 5
+  },
+  {
+    label: '5:30 am',
+    value: 5.5
+  },
+  {
+    label: '6:00 am',
+    value: 6
+  },
+  {
+    label: '7:30 am',
+    value: 7.5
+  },
+  {
+    label: '8:00 am',
+    value: 8.0
+  },
+  {
+    label: '8:30 am',
+    value: 8.5
+  },
+  {
+    label: '9:00 am',
+    value: 9
+  },
+  {
+    label: '9:30 am',
+    value: 9.5
+  },
+  {
+    label: '10:00 am',
+    value: 10
+  },
+  {
+    label: '10:30 am',
+    value: 10.5
+  },
+  {
+    label: '11:00 am',
+    value: 11
+  },
+  {
+    label: '11:30 am',
+    value: 11.5
+  },
+  {
+    label: '12:00 pm',
+    value: 12.0
+  },
+  {
+    label: '12:30 pm',
+    value: 12.5
+  },
+  {
+    label: '1:00 pm',
+    value: 13
+  },
+  {
+    label: '1:30 pm',
+    value: 13.5
+  },
+  {
+    label: '2:00 pm',
+    value: 14
+  },
+  {
+    label: '2:30 pm',
+    value: 14.5
+  },
+  {
+    label: '3:00 pm',
+    value: 15
+  },
+  {
+    label: '3:30 pm',
+    value: 15.5
+  },
+  {
+    label: '4:00 pm',
+    value: 16.0
+  },
+  {
+    label: '4:30 pm',
+    value: 16.5
+  },
+  {
+    label: '5:00 pm',
+    value: 17.0
+  },
+  {
+    label: '5:30 pm',
+    value: 17.5
+  },
+  {
+    label: '6:00 pm',
+    value: 18
+  },
+  {
+    label: '6:30 pm',
+    value: 18.5
+  },
+  {
+    label: '7:00 pm',
+    value: 19
+  },
+  {
+    label: '7:30 pm',
+    value: 19.5
+  },
+  {
+    label: '8:00 pm',
+    value: 20
+  },
+  {
+    label: '8:30 pm',
+    value: 20.5
+  },
+  {
+    label: '9:00 pm',
+    value: 21.0
+  },
+  {
+    label: '9:30 pm',
+    value: 21.5
+  },
+  {
+    label: '10:00 pm',
+    value: 22.0
+  },
+  {
+    label: '10:30 pm',
+    value: 22.5
+  },
+  {
+    label: '11:00 pm',
+    value: 23
+  },
+  {
+    label: '11:30 pm',
+    value: 23.5
+  }
```

Comment:
should people be able to be available until 11:59 pm?

Reply:
Good call.  I'll add that.   I assume 23.99 is the right number to send to the DB.

Reply:
done